### PR TITLE
Ignore XamlGeneratedNamespace.GeneratedInternalTypeHelper

### PR DESCRIPTION
**Pull request by tmat**
Workaround for https://github.com/dotnet/wpf/issues/2690

### CODE REVIEW

File: src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs [hunk 1]
```diff
@@ -339,6 +339,13 @@ void reportDiagnosticAtLocations(DiagnosticDescriptor descriptor, ImmutableDicti
 
                 void reportDeclareNewApi(ISymbol symbol, bool isImplicitlyDeclaredConstructor, string publicApiName)
                 {
+                    // TODO: workaround for https://github.com/dotnet/wpf/issues/2690
```
Comment (mavasani) [inline src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs]: Do we need to file a tracking issue in roslyn-analyzers repo to revert this change once https://github.com/dotnet/wpf/issues/2690 is fixed? If not, can you add a comment to https://github.com/dotnet/wpf/issues/2690 that this workaround needs to be reverted once that issue is fixed?

Comment (tmat) [reply src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs]: https://github.com/dotnet/roslyn-analyzers/issues/3345
